lib/plugins/aws/package/compile/events/schedule.js

```diff
@@ -68,16 +69,60 @@ class AwsCompileScheduledEvents {
               },
               required: ['inputTemplate'],
               additionalProperties: false,
+            }
```

Comment:
Diff needs to be cleaned up to only include changed properties

Reply:
I'm sorry but I don't understand this. To explain what I've done with the json-schema. I've created two different schemas for schedule, one for eventbus schedule which stays as the old schema and a new one for scheduler which adds new properties -> method, roleArn, timezone and removes inputPath and inputTransformer.
Doing it this way will ensure that people don't mix the two and schema check will detect and let them know that these properties are exclusive and cannot be used together, this is why method is required in the second schema.
I've added title to prevent confusion.